docs/source/index.rst

```diff
@@ -1,4 +1,4 @@
-.. echopype documentation master file, created by
+OA.. echopype documentation master file, created by
```

Comment:
What's this "OA" string? It seems to be changing the directive that effectively commented out this block. Now the block is being displayed, which seems undesirable.

Reply:
It is an artifact of Emacs. Sorry about that! I will make sure this does not happen in the future.